### PR TITLE
[PhpUnitBridge] Add polyfill for PhpUnit namespace

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? |no
| Tests pass?   | yes
| Fixed tickets | #32844
| License       | MIT
| Doc PR        | NA

This PR provides PhpUnit >= 6 Namespaces class.

This will simplify tests that have to be compatible with multiple version of PHPUnit

```diff
- if (class_exists('PHPUnit_Foo')) {
-   PHPUnit_Foo::bar():
- } else {
-   \PHPUnit\Foo::bar();
- }
+ \PHPUnit\Foo::bar();
```

WIP  => waiting for #32941 to be green

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/bootstrap.php [hunk 1]
```diff
@@ -12,6 +12,95 @@
...
+        'PHPUnit_Runner_BaseTestRunner',
+        'PHPUnit_Runner_Version',
+
+        'PHPUnit_Util_Blacklist',
+        'PHPUnit_Util_ErrorHandler',
+        'PHPUnit_Util_Test',
+        'PHPUnit_Util_XML',
+    ];
+    foreach ($classes as $class) {
+        class_alias($class, '\\'.strtr($class, ['_' => '\\']), true);
```
Comment (jderusse) [inline src/Symfony/Bridge/PhpUnit/bootstrap.php]: Just wonder for a performance point of view (opcode). Is it better to remove variables? Ie.
```php
class_alias('A_Foo', '\A\Foo);
class_alias('A_Bar', '\A\Bar);
```

Comment (nicolas-grekas) [reply src/Symfony/Bridge/PhpUnit/bootstrap.php]: `class_alias($class, strtr($class, '_', '\\'));` would be fine to me

Comment (jderusse) [reply src/Symfony/Bridge/PhpUnit/bootstrap.php]: fixed `['_' => '\\']` replaced by `'_', '\\'`

File: src/Symfony/Bridge/PhpUnit/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * made the bridge act as a polyfill for newest PHPUnit features
  * added `SetUpTearDownTrait` to allow working around the `void` return-type added by PHPUnit 8
+ * added namespace aliases for phpunit < 6
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/CHANGELOG.md]: s/phpunit/PHPUnit

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @jderusse.

**Review by fabpot — APPROVED** [review]

**Pull request merged**